aip/general/0008.md

```diff
@@ -38,10 +38,9 @@ AIPs **must** have appropriate front matter.
 
 ```yaml
 ---
-aip:
-  id: 8
-  state: reviewing
-  created: 2019-05-28
+id: 8
+state: reviewing
```

Comment:
I think this should be approved? The site has it as approved anyways...weird
![image](https://github.com/aip-dev/google.aip.dev/assets/6644735/015b02d2-ae5a-4a7d-aa8c-3f9686b49034)